### PR TITLE
Use Gatsby theme

### PR DESCRIPTION
**Pull request by okonet**
Uses https://github.com/component-driven/gatsby-theme-content-collections

### CODE REVIEW

File: src/components/Img.js [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
-        maxWidth: "100%",
-        ...css,
-      }}
-      {...rest}
-    />
-  )
+const StyledImage = styled.img({
+  display: "inline-block",
+  maxWidth: "100%",
+})
```
Comment (okonet) [inline src/components/Img.js:7]: Not sure why but it started to fail on `css` prop...

File: src/components/Img.js [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
+
+function Img({ src, altText, ...rest }) {
+  return <StyledImage src={src} alt={altText} {...rest} />
```
Comment (okonet) [inline src/components/Img.js:10]: Can be simplified...

File: src/components/OfferingPost.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { MDXRenderer } from "gatsby-plugin-mdx"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+import Section from "../components/Section"
+import Heading from "../components/ui/Heading"
+
+function OfferingPost({ data, location }) {
+  const { post } = data
+  return (
+    <Layout location={location} title={"siteTitle"}>
```
Comment (okonet) [inline src/components/OfferingPost.js:11]: oops `siteTitle` → `title`

File: src/components/OfferingPosts.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+          </Heading>
+          <Text>
+            We offer one-day and multi-day workshops as well as guidance and
+            leadership services covering component-driven development, design
+            systems, styleguides & pattern libraries, and modern frontend
+            architecture.
+          </Text>
+          <ol>
+            {allPost.nodes.map(post => (
+              <Link to={post.slug} key={post.slug}>
```
Comment (sapegin) [inline src/components/OfferingPosts.js:28]: Looks like you have `li` inside `a` here.

File: src/components/OfferingPosts.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                <Heading size="m" as="li" css={{ marginBottom: "1rem" }}>
```
Comment (sapegin) [inline src/components/OfferingPosts.js:29]: Don't you have `mb` prop here?

File: src/components/OfferingPosts.js (hunk 4 above)
Comment (okonet) [reply src/components/OfferingPosts.js:28]: Oops

**Pull request merged**